config/purposes/bespoke_chromium_3pv3.yml

```diff
@@ -11,21 +11,21 @@ LBC 3pV3 GEX Dil:
   - limber_chromium_bespoke
   :expected_library_types:
   - Chromium single cell 3 prime v3
-  :concentration_binning:
+  :dilutions:
     :source_volume: 10
     :diluent_volume: 25
     :bins:
     - colour: 1
       pcr_cycles: 16
-      max: 25
+      max: 24
```

Comment:
Should be 25 here, cover with thre dots eg. (1...12) covers everything between 1 and 12, excluding 12 itself, but including, say 11.9999999